{"name": "test_database_transaction_fixture", "status": "broken", "statusDetails": {"message": "sqlalchemy.exc.ArgumentError: Textual SQL expression 'INSERT INTO test_table (n...' should be explicitly declared as text('INSERT INTO test_table (n...')", "trace": "testcase\\test_orm_integration.py:271: in test_database_transaction_fixture\n    db_transaction.execute(sql, test_data)\n.venv\\lib\\site-packages\\sqlalchemy\\orm\\session.py:2365: in execute\n    return self._execute_internal(\n.venv\\lib\\site-packages\\sqlalchemy\\orm\\session.py:2149: in _execute_internal\n    statement = coercions.expect(roles.StatementRole, statement)\n.venv\\lib\\site-packages\\sqlalchemy\\sql\\coercions.py:395: in expect\n    resolved = impl._literal_coercion(\n.venv\\lib\\site-packages\\sqlalchemy\\sql\\coercions.py:621: in _literal_coercion\n    return self._text_coercion(element, argname, **kw)\n.venv\\lib\\site-packages\\sqlalchemy\\sql\\coercions.py:614: in _text_coercion\n    return _no_text_coercion(element, argname)\n.venv\\lib\\site-packages\\sqlalchemy\\sql\\coercions.py:584: in _no_text_coercion\n    raise exc_cls(\nE   sqlalchemy.exc.ArgumentError: Textual SQL expression 'INSERT INTO test_table (n...' should be explicitly declared as text('INSERT INTO test_table (n...')"}, "description": "测试数据库事务fixture", "steps": [{"name": "测试设置: TestDatabaseFixtures.test_database_transaction_fixture", "status": "passed", "start": 1754271683194, "stop": 1754271683196}, {"name": "测试用例: 数据库事务测试", "status": "broken", "statusDetails": {"message": "sqlalchemy.exc.ArgumentError: Textual SQL expression 'INSERT INTO test_table (n...' should be explicitly declared as text('INSERT INTO test_table (n...')\n", "trace": "  File \"C:\\Users\\sy_li\\bgtest\\utils\\allure_decorators.py\", line 31, in wrapper\n    result = func(*args, **kwargs)\n  File \"C:\\Users\\sy_li\\bgtest\\utils\\test_decorators.py\", line 29, in wrapper\n    return func(*args, **kwargs)\n  File \"C:\\Users\\sy_li\\bgtest\\utils\\test_decorators.py\", line 286, in wrapper\n    return func(*args, **kwargs)\n  File \"C:\\Users\\sy_li\\bgtest\\utils\\test_decorators.py\", line 300, in wrapper\n    return func(*args, **kwargs)\n  File \"C:\\Users\\sy_li\\bgtest\\testcase\\test_orm_integration.py\", line 271, in test_database_transaction_fixture\n    db_transaction.execute(sql, test_data)\n  File \"C:\\Users\\sy_li\\bgtest\\.venv\\lib\\site-packages\\sqlalchemy\\orm\\session.py\", line 2365, in execute\n    return self._execute_internal(\n  File \"C:\\Users\\sy_li\\bgtest\\.venv\\lib\\site-packages\\sqlalchemy\\orm\\session.py\", line 2149, in _execute_internal\n    statement = coercions.expect(roles.StatementRole, statement)\n  File \"C:\\Users\\sy_li\\bgtest\\.venv\\lib\\site-packages\\sqlalchemy\\sql\\coercions.py\", line 395, in expect\n    resolved = impl._literal_coercion(\n  File \"C:\\Users\\sy_li\\bgtest\\.venv\\lib\\site-packages\\sqlalchemy\\sql\\coercions.py\", line 621, in _literal_coercion\n    return self._text_coercion(element, argname, **kw)\n  File \"C:\\Users\\sy_li\\bgtest\\.venv\\lib\\site-packages\\sqlalchemy\\sql\\coercions.py\", line 614, in _text_coercion\n    return _no_text_coercion(element, argname)\n  File \"C:\\Users\\sy_li\\bgtest\\.venv\\lib\\site-packages\\sqlalchemy\\sql\\coercions.py\", line 584, in _no_text_coercion\n    raise exc_cls(\n"}, "attachments": [{"name": "测试用例异常: 数据库事务测试", "source": "7db991a5-d79c-4cf7-ac99-1c15a87e446b-attachment.json", "type": "application/json"}], "start": 1754271683196, "stop": 1754271683198}, {"name": "测试清理: TestDatabaseFixtures.test_database_transaction_fixture", "status": "passed", "start": 1754271683383, "stop": 1754271683387}], "attachments": [{"name": "测试异常", "source": "e2e454f2-7add-4bcd-a26c-6f1869e53200-attachment.json", "type": "application/json"}, {"name": "异常堆栈", "source": "7381852d-cb5b-450a-b073-37dfd44e42af-attachment.txt", "type": "text/plain"}, {"name": "log", "source": "86453825-4114-407d-9476-bc9d0388dbd7-attachment.txt", "type": "text/plain"}], "start": 1754271683196, "stop": 1754271683198, "uuid": "f79836ff-6234-4610-80c9-2e878137410f", "historyId": "6f7034cfea90060ef04762027a14cae3", "testCaseId": "6f7034cfea90060ef04762027a14cae3", "fullName": "testcase.test_orm_integration.TestDatabaseFixtures#test_database_transaction_fixture", "labels": [{"name": "story", "value": "事务管理"}, {"name": "feature", "value": "Fixtures"}, {"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_orm_integration"}, {"name": "subSuite", "value": "TestDatabaseFixtures"}, {"name": "host", "value": "lihaojie"}, {"name": "thread", "value": "89128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_orm_integration"}], "titlePath": ["testcase", "test_orm_integration.py", "TestDatabaseFixtures"]}